src/Nox.Generator/Domain/Entity.template.cs

```diff
@@ -46,6 +46,9 @@ public partial class {{className}}{{ if !entity.IsOwnedEntity }} : {{if entity.P
 		}
 	}
 
+	{{- else if key.Type == "Guid" -}}
```

Comment:
I don't think you need to handle Guid type specifically, because it would generate the "else" section.
You would probably just need to add "Nox.Types" namespace before the "{{key.Type}}" on line 54.

Reply:
Desired changes were applied